README.md

```diff
@@ -302,25 +282,25 @@ Two storage adapters are currently supported: Git and MongoDB. Each one can be u
 
 #### Environment variables
 
-Environment variables can be provided in a `.env` file at the root of the repository. See `.env.example` for an example of such a file.
+Environment variables can be passed in the command-line or provided in a `.env` file at the root of the repository. See `.env.example` for an example of such a file.
```

Comment:
Perhaps add a note to explain that if we add `NODE_ENV` to the `.env`, it will not work. It will only work for the variables defined below, not for all environment variables.